js/filter.js

```diff
@@ -0,0 +1,61 @@
+import { debounce } from './util.js';
+import { renderThumbnails } from './thumbnails.js';
+
+let currentFilter = 'filter-default';
+let pictures = [];
+const filterElement = document.querySelector('.img-filters');
+const ACTIVE_BUTTON_CLASS = 'img-filters__button--active';
+
+const debounceRender = debounce(renderThumbnails);
+
+const FILTER = {
+  default: 'filter-default',
+  random: 'filter-random',
+  discussed: 'filter-discussed',
+};
```

Comment:
[Критерий Б8](https://up.htmlacademy.ru/javascript-individual/2/criteries#b8)

```suggestion
const Filter = {
  DEFAULT: 'filter-default',
  RANDOM: 'filter-random',
  DISCUSSED: 'filter-discussed',
};
```